.github/workflows/full-test-suite.yml

```diff
@@ -2,8 +2,18 @@ name: dice-test-suite
 on:
   push:
     branches: [master]
+    paths:
+      - "**/*.go"
+      - "go.mod"
```

Comment:
Can we have it other way around like a blacklist of paths instead of whitelist?

Reply:
Hey @lucifercr07, I have switched to having the paths blacklist.

**Ref:** https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore